state-chain/custom-rpc/src/lib.rs

```diff
@@ -312,12 +322,6 @@ pub trait CustomApi {
 		account_id: state_chain_runtime::AccountId,
 		at: Option<state_chain_runtime::Hash>,
 	) -> RpcResult<RpcAccountInfo>;
-	#[method(name = "account_info_v2")]
-	fn cf_account_info_v2(
-		&self,
-		account_id: state_chain_runtime::AccountId,
-		at: Option<state_chain_runtime::Hash>,
-	) -> RpcResult<RpcAccountInfoV2>;
```

Comment:
it's a breaking change but it's a good time to do it with 1.3 coming soon and this old one being deprecated and only used for getting validator info